modules/loader/src/main/java/com/opengamma/strata/loader/fpml/FpmlDocumentParser.java

```diff
@@ -195,29 +196,60 @@ private FpmlDocumentParser(
 
   // intelligently finds the FpML root element
   private static XmlElement findFpmlRoot(XmlElement root) {
-    if (isFpmlRoot(root)) {
-      return root;
+    XmlElement fpmlRoot = isFpmlRoot(root);
```

Comment:
Since `isFpmlRoot()` doesn't return a boolean anymore, is it worth renaming it to something more appropriate, such as `getFpmlRoot()`?